src/dra/dra.h

```diff
@@ -215,6 +215,21 @@ typedef struct {
     u32 unk8;
 } DamageParam;
 
+typedef struct {
+    /* 8013761C */ MenuContext menus[3];
+    /* 80137676 */ s16 unkPad1;
```

Comment:
Ah yes, I just called them "pad" because I didn't know what they were for and I was using them as padding to build the struct - I don't expect any of them to actually be padding and all will likely have some use somewhere.

I will give them all D_ names.

Reply:
On second thought, decided to call them unused. If we go through the whole decomp and get everything matching and those are still completely untouched, that will be a good thing to be aware of.